php/templates/deployment.yaml

```diff
@@ -245,13 +242,17 @@ spec:
 {{- with .Values.extraVolumes }}
 {{ tpl (toYaml .) $root | indent 6 }}
 {{- end }}
+{{- with .Values.restartPolicy }}
+      restartPolicy: {{ . }}
+{{- end }}
 {{- with .Values.imagePullSecrets}}
       imagePullSecrets:
 {{ tpl (toYaml .) $root | indent 6 }}
 {{- end }}
 {{- with .Values.serviceAccountName }}
       serviceAccountName: {{ . }}
 {{- end }}
+      terminationGracePeriodSeconds: {{ add . 10 }}
```

Comment:
What would it add 10 to...?

Reply:
@cw-sakamoto 
Sorry, I fixed it.
https://github.com/chatwork/charts/pull/117/commits/7fd582e7a5ad3e99f6227c1e183fc386878ac9de